analytics/analytics-ui/src/app/table/table.component.html

```diff
@@ -1,9 +1,20 @@
-<mat-table [dataSource]="dataSource" class="mat-elevation-z8" matSort>
-  <ng-container [matColumnDef]="column" *ngFor="let column of tableColumns">
-    <mat-header-cell *matHeaderCellDef mat-sort-header>{{column}}</mat-header-cell>
-    <mat-cell *matCellDef="let element">{{element[column]}}</mat-cell>
-  </ng-container>
+<div style="margin-left:110px">
+  <h1>Table</h1>
+</div>
 
-  <mat-header-row *matHeaderRowDef="tableColumns; sticky: true"></mat-header-row>
-  <mat-row *matRowDef="let row; columns: tableColumns;"></mat-row>
-</mat-table>
\ No newline at end of file
+<!-- Dynamic Table -->
+
+<mat-card class="mat-elevation-z8">
```

Comment:
This leaves a gap between the edge of the card and the table which looks a bit odd. Can the card wrapping be removed? It will still follow the same design principles as the rest of the app since Angular material wraps it in a card anyway.